[Bug]: GroupChecker unit test fail occasinally

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: a568fae
- Deployment mode(standalone or cluster): both
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

Fail like
```
--- FAIL: TestGroupChecker (0.02s)
    group_checker_test.go:41: 
        	Error Trace:	group_checker_test.go:41
        	Error:      	Not equal: 
        	            	expected: []string{"1", "2"}
        	            	actual  : []string{"2", "1"}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,4 +1,4 @@
        	            	 ([]string) (len=2) {
        	            	- (string) (len=1) "1",
        	            	- (string) (len=1) "2"
        	            	+ (string) (len=1) "2",
        	            	+ (string) (len=1) "1"
        	            	 }
        	Test:       	TestGroupChecker
FAIL
exit status 1

```

### Expected Behavior

Unit test passes

### Steps To Reproduce

```markdown
Run unittest
```


### Anything else?

_No response_

--- a/internal/util/timerecord/group_checker_test.go
+++ b/internal/util/timerecord/group_checker_test.go
@@ -38,12 +38,12 @@ func TestGroupChecker(t *testing.T) {
 	assert.Equal(t, 10*time.Millisecond, gc2.d)
 
 	list := <-signal
-	assert.Equal(t, []string{"1", "2"}, list)
+	assert.ElementsMatch(t, []string{"1", "2"}, list)
 
 	gc2.Remove("2")
 
 	list = <-signal
-	assert.Equal(t, []string{"1"}, list)
+	assert.ElementsMatch(t, []string{"1"}, list)
 
 	assert.NotPanics(t, func() {
 		gc1.Stop()

